package.json

```diff
@@ -8,7 +8,7 @@
   "author": "Luigi Pinca",
   "license": "MIT",
   "scripts": {
-    "minify": "for f in public/js/{app,home,leaderboards}; do uglifyjs $f.js -o $f.min.js; done",
+    "minify": "for f in *.js; do short=${f%.js}; uglifyjs $f > $short.min.js; done",
```

Comment:
This is wrong as `*.js` expands to all `.js` files in the current directory.

The problem with this script is that `npm run-script` always uses `/bin/sh` which is dash on Ubuntu and not bash so there is no support for brace expansion.

Can you update the script as follows?

```
for f in app home leaderboards; do f=public/js/$f; uglifyjs $f.js -o $f.min.js; done
```

This should fix the issue.